### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by literarymachine**
Travis CI builds the site and pushes it back to the `pages-production` or `pages-staging` branch respectively.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- git config user.email ${EMAIL}
+- rsync -az --delete --exclude '.git*' ../_site/ .
+- touch .nojekyll
+- git add -A .
+- git commit -m "Generated Jekyll site by Travis CI - ${TRAVIS_BUILD_NUMBER}"
+- ! '[ "${TRAVIS_BRANCH}" = "master" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && git push --force "https://${DEPLOY_KEY}@${GITHUB_REPO}" HEAD:${PRODUCTION_TARGET_BRANCH}'
+- ! '[ "${TRAVIS_BRANCH}" = "master" ] && [ "${TRAVIS_PULL_REQUEST}" != "false" ] && git push --force "https://${DEPLOY_KEY}@${GITHUB_REPO}" HEAD:${STAGING_TARGET_BRANCH}'
+env:
+  global:
+  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
```
Comment (anarchivist) [inline .travis.yml]: Should this be `  - NOKOGIRI_USE_SYSTEM_LIBRARIES: true`

**Comment by literarymachine** [comment]
We should be all set for this once

- [ ] [Build only if .travis.yml is present](https://travis-ci.org/rightsstatements/rightsstatements.github.io/settings) is activated,
- [ ] `.travis.yml` is updated with secure `EMAIL` and `DEPLOY_KEY` as described [here](http://nick-dunn.co.uk/blog/jekyll-github-travis-ci-sitting-in-a-tree/) and
- [ ] `GITHUB_REPO` in `.travis.yml` is updated to `github.com/rightsstatements/rightsstatements.github.io.git`.

After that, we should create a pull request to master to create the `pages-staging` branch and merge it to create the `pages-production` branch.

**Comment by anarchivist** [comment]
@literarymachine I've got the values you'd need for `.travis.yml` for `EMAIL` and `DEPLOY_KEY`. How would you like me to add them?

**Comment by anarchivist** [comment]
(I've also changed the Travis settings to enable `Build only if .travis.yml is present`.)

**Comment by literarymachine** [comment]
@anarchivist Perfect! The best way to hand them over would be to use the [Travis command line client](https://blog.travis-ci.com/2013-01-14-new-client/) from within your clone of this repo to encrypt them:

$ travis encrypt DEPLOY_KEY=foobar
$ travis encrypt EMAIL=foo@example.org

If that works, you can just send me the results by mail. Thanks!

**Comment by anarchivist** [comment]
Sent via email, thanks!

**Comment by anarchivist** [comment]
OK, here we go...

**Pull request merged**